examples/file-read-buffered.roc

```diff
@@ -23,20 +23,13 @@ main =
     readSummary = Task.loop!
         { linesRead: 0, bytesRead: 0 }
         (processLine reader)
-
-    Stdout.line! "Done reading, got $(Inspect.toStr readSummary)"
-
-    # If we close a reader, we will get EOF for any future reads.
-    closedReader = File.getFileReader! "README.md"
-
-    File.closeFileReader! closedReader
-
-    when File.readLine closedReader |> Task.result! is
-        Ok bytes if List.len bytes == 0 -> Stdout.line "Got EOF reading from closed file."
-        _ -> crash "unexpected; we should get an EOF trying to read a file after closing"
```

Comment:
What do you think about this change @lukewilliamboswell?
I feel like the examples should briefly demonstrate best practices, explaining this edge case does not seem necessary.